components/motor/gpiostepper/gpiostepper.go

```diff
@@ -337,7 +341,7 @@ func (m *gpioStepper) GoTillStop(ctx context.Context, rpm float64, stopFunc func
 	}
 	defer func() {
 		if err := m.Stop(ctx, nil); err != nil {
-			m.logger.Errorw("failed to turn off motor", "error", err)
+			m.logger.Errorw("failed to turn off motor (%s)", m.motorName, "error", err)
```

Comment:
I'm having trouble finding the definition of `Errorw`, but I suspect that it needs an odd number of arguments: the main string to log, and then a series of key/value pairs. I don't think it supports formatting like this.

Reply:
Maybe you meant to have a `fmt.Sprintf` call in the middle? I think that would do the right thing.

Reply:
Yes, and this Errorw is part of the logger package.